### PR TITLE
Virginia Tech edits to 2021 annual report

### PR DESCRIPTION
**Pull request by ksterne**
Just a typo and capture of activity during the workshop.  Wanted to start this branch and pull request to ask a question as well.

### CODE REVIEW

File: reports/Krieger_ddwg_2021.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ servers are expected to move to a more permanent home in the future, but the dat
 1. File issues are being tracked down as part of regular maintenance.
 1. Some time is being spent in trying to understand changes in data distribution that have not been communicated to the WG. 
 Explicit, direct and clear communication will be appreciated regarding changes.
-1. A future TODO is to prioritize setting Paul Breen up with an account so that BAS can receive vks, fhe, fhw, gbr and kap data.
+1. A future TODO is to prioritize setting Paul Breen up with an account so that BAS can receive bks, fhe, fhw, gbr and kap data. Initial setup of BAS account was done during SD Workshop.
 1. FITACF 3.0 files no longer shared via Globus
```
Comment (ksterne) [inline reports/Krieger_ddwg_2021.md:19]: Did we ever distribute fitacf3 files to USask and globus?  I'm very confused by this comment.

Comment (kevinkrieger) [reply reports/Krieger_ddwg_2021.md:19]: Yes we (usask) did briefly due to some requests, but then we stopped due to the SuperDARN community not having consensus on FITACF 3 yet. We have FITACF 2.5 files being shared right now instead.

**Review by kevinkrieger — APPROVED** [review]

File: reports/Krieger_ddwg_2021.md (hunk 1 above)
Comment (ksterne) [reply reports/Krieger_ddwg_2021.md:19]: I know you've got this merged in, but thought I would continue here.  Should the update for sharing fitacf 3.0 files then be under updates for USask and not under updates for VT?

Comment (kevinkrieger) [reply reports/Krieger_ddwg_2021.md:19]: Thanks for catching that, updated now!

**Pull request merged**